zk_watcher.go

```diff
@@ -137,6 +137,8 @@ func (w *zkWatcher) cancelWatches() {
 	}
 }
 
+const waitPeriod = 10 * time.Second
```

Comment:
this is a const in the whole package, so it should be named more specifically. perhaps `zkReconnectPeriod` or similar?